testing/testcompat/v203/com/google/idea/sdkcompat/ComponentManagerWrapper.java

```diff
@@ -0,0 +1,19 @@
+package com.google.idea.sdkcompat;
+
+import org.picocontainer.ComponentAdapter;
+import org.picocontainer.MutablePicoContainer;
+
+import com.intellij.openapi.components.ComponentManager;
+
+// #api212
```

Comment:
Here and for the other classes: Please add a short explanation on what to do when addressing the #api212 tag in the future.